app.py

```diff
@@ -19,11 +24,21 @@ def root():
 @app.route("/upload", methods=["GET", "POST"])
 def upload():
     if request.method == "POST":
+        uploads = {"success": [], "failed": []}
```

Comment:
these are the results of the uploads, name it differently?

Reply:
renamed to `file_upload_status`